plugins/base/src/main/kotlin/translators/psi/parsers/PsiCommentsUtils.kt

```diff
@@ -98,7 +98,7 @@ internal fun findClosestDocComment(element: PsiNamedElement, logger: DokkaLogger
             return findClosestDocComment(superMethods.single(), logger)
         }
 
-        val superMethodDocumentation = superMethods.map { method -> findClosestDocComment(method, logger) }
+        val superMethodDocumentation = superMethods.map { method -> findClosestDocComment(method, logger) }.distinct()
```

Comment:
Without `distinct`, the below `if` condition was pointless since the arity of the list won't change while `mapping` and we are testing 3 lines earlier that list is not a single value.